README.rst

```diff
@@ -47,7 +55,7 @@ It doesn't work
 
 EIN is tested on GNU Emacs versions
 25.1
-and later. Your mileage may vary with the `spacemacs layer`_ and other *emacsen*.
+and later.  We presently do not recommend the `spacemacs layer`_.
```

Comment:
No changes needed, but it is ironic considering my dogged use of spacemacs. I do agree there are a couple issues that spacemacs introduces:

 1. The dependency on `undo-tree` that is breaking ein's undo system. `undo-tree` is tightly integrated with `evil` so that is a problem.
 2. Spacemacs concept of leader keys breaks in poly mode, as each different mode has its own set of bindings. I'm looking at ways around this, if I find one I will probably add it as a hint in the wiki.